packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.test.js

```diff
@@ -72,6 +72,20 @@ describe('ListToVirtualizedList', () => {
 		expect(table.rowHeight).toBe(200);
 	});
 
+	it('should support disableSort', () => {
```

Comment:
Please state in test description that headers that are hidden due to `hideHeader: true` should have the prop disableSort set to true to reflect your test code